packages/discovery-react-components/src/components/DocumentPreview/components/PdfHighlight/types.ts

```diff
@@ -20,6 +20,8 @@ export type DocumentFieldHighlight = {
   fieldIndex: number;
   location: Location;
   className?: string;
+  facetId?: string;
+  // TODO: add category?
```

Comment:
Is this TODO something you still need to address?

Reply:
thanks, removed